lib/stb_image.ex

```diff
@@ -103,6 +103,105 @@ defmodule StbImage do
         "unsupported tensor shape: #{inspect(shape)} (expected height-width-channel)"
       )
 
+  if Code.ensure_loaded?(Kino.Render) do
+    defimpl Kino.Render do
+      defp within_maximum_size(image) do
+        max_size = Application.fetch_env!(:stb_image, :kino_render_max_size)
+
+        case max_size do
+          {max_height, max_width} when is_integer(max_height) and is_integer(max_width) ->
+            case image.shape do
+              {h, w} ->
+                h <= max_height and w <= max_width
```

Comment:
Shouldn't we always have channels in the tuple?

Reply:
Yup, I’ll remove this one😃